lib/video_info/providers/aparat.rb

```diff
@@ -0,0 +1,137 @@
+class VideoInfo
+  module Providers
+    class Aparat < Provider
+      def self.usable?(url)
+        url =~ /((.*)aparat\.com\/v\/(.*))/
+      end
+
+      def provider
+        'Aparat'
+      end
+
+      def api_key
+        ""
+      end
+
+      %w[description].each do |method|
+        define_method(method) { _video[method] }
+      end
+
+      %w[duration width height].each do |method|
+        define_method(method) { _video[method].to_i }
+      end
+
+      def title
+        _video['title']
+      end
+
+      def author
+        _video['username']
+      end
+
+      def author_thumbnail_id
+        author_uri = _video['username']
+        @author_thumbnail_id ||= _parse_picture_id(author_uri)
+      end
+
+      def author_thumbnail(width = 75)
+        _video['profilePhoto']
+      end
+
+      def thumbnail_id
+        @thumbnail_id = "123"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.